Fabric.Identity.API/Quickstart/Account/ClaimsResult.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.

```

Comment:
is there any reason this is not in models?
looks like it was copied in from another project / base?

Reply:
The refactoring started in that namespace.

While I like the Models folder for this, I feel the intent for Models folder were Models that were external facing types.  I dont  see this being used externally, I dont see alternatives to where this could go.